applications/MedApplication/med_inc.h

```diff
@@ -14,8 +14,8 @@
 
 // Those dummy structs are necessary when the MED-library is compiled with MPI support.
 // Then the header also contains functions which use those MPI-types
-struct MPI_Comm{};
-struct MPI_Info{};
+//struct MPI_Comm;
+//struct MPI_Info;
```

Comment:
the issue here is when you compile without MPI

I am not sure how we can go about this. Maybe `ifdef`? Might be the only option we have for now, can you give it a try?

Also, how did you install the med-lib? By hand or via the package manager?

Reply:
> Maybe ifdef?

Yep, I think we'll have to resort to the preprocessor.

> Also, how did you install the med-lib? By hand or via the package manager?

I installed [salome-medcoupling](https://aur.archlinux.org/packages/salome-medcoupling) (should include both the serial and OpenMPI med lib) using the system package manager.

Reply:
Ok ifdef it is (at least for now, until someone complains :D), can you please add it?

Also, can you please add how you installed the lib to the readme? I did it already for ubuntu